node-registrar/cmds/main.go

```diff
@@ -0,0 +1,74 @@
+package cmds
+
+import (
+	"errors"
+	"flag"
+	"fmt"
+
+	"github.com/rs/zerolog"
+	"github.com/rs/zerolog/log"
+	"github.com/threefoldtech/tfgrid-sdk-go/node-registrar/pkg/db"
+	"github.com/threefoldtech/tfgrid-sdk-go/node-registrar/pkg/server"
+	"gorm.io/gorm/logger"
+)
+
+type flags struct {
+	db.Config
+	debug   bool
+	version bool
+	domain  string
+	port    int
+}
+
+var (
+	commit  string
+	version string
+)
+
+func Run() {
+	f := flags{}
+	var sqlLogLevel int
+	flag.StringVar(&f.Host, "postgres-host", "", "postgres host")
+	flag.IntVar(&f.Config.Port, "postgres-port", 5432, "postgres port")
+	flag.StringVar(&f.DBName, "postgres-db", "", "postgres database")
+	flag.StringVar(&f.User, "postgres-user", "", "postgres username")
+	flag.StringVar(&f.Password, "postgres-password", "", "postgres password")
+	flag.StringVar(&f.SSLMode, "ssl-mode", "disable", "postgres ssl mode[disable, require, verify-ca, verify-full]")
+	flag.IntVar(&sqlLogLevel, "sql-log-level", 2, "sql logger level")
+
+	flag.BoolVar(&f.version, "v", false, "shows the package version")
+	flag.BoolVar(&f.debug, "debug", false, "allow debug logs")
+	flag.IntVar(&f.port, "server-port", 443, "server port")
+	flag.StringVar(&f.domain, "domain", "", "domain on which the server will be served")
+
+	flag.Parse()
+	f.SqlLogLevel = logger.LogLevel(sqlLogLevel)
```

Comment:
Do bound check on the loglevel before setting it https://github.com/go-gorm/gorm/blob/master/logger/logger.go#L39